File: Python/백준/Bronze/1871. 좋은 자동차 번호판/좋은 자동차 번호판.py
a = int(input())
for i in range(a):
    b,c = input().split('-')
    s1 = 0
    s2 = int(c)
    for j in range(3):
        s1+=(ord(b[j])-65)*(26**(2-j))
    if abs(s1-s2)<=100:
        print('nice')
    else:
        print('not nice')

File: C++11/백준/Bronze/1871. 좋은 자동차 번호판/좋은 자동차 번호판.cc
#include <stdio.h>
#include <cmath>

int main(){
    int a,n2;
    char c1,c2,c3,c4;
    scanf("%d\n",&a);
    for(int i=0;i<a;i++){
        scanf("%c%c%c%c%d\n",&c1,&c2,&c3,&c4,&n2);
        int n1 = (c1-'A')*26*26+(c2-'A')*26+(c3-'A');
        if(abs(n1-n2)<=100){
            printf("nice\n");
        }
        else{
            printf("not nice\n");
        }
    }
    return 0;
}